apps/web/src/components/NextRoleAndCareerObjective/NextRoleDialog.tsx

```diff
@@ -0,0 +1,73 @@
+import { ReactNode, useState } from "react";
+import { useIntl } from "react-intl";
+
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { Button, Dialog } from "@gc-digital-talent/ui";
+
+import NextRole from "./NextRole";
+
+export const NextRoleDialog_Fragment = graphql(/* GraphQL */ `
+  fragment NextRoleDialog on User {
+    firstName
+    employeeProfile {
+      ...NextRoleInfo
+    }
+  }
+`);
+
+interface NextRoleDialogProps {
+  nextRoleDialogQuery: FragmentType<typeof NextRoleDialog_Fragment>;
+  trigger?: ReactNode;
+  defaultOpen?: boolean;
+}
+
+const NextRoleDialog = ({
+  nextRoleDialogQuery,
+  trigger,
+  defaultOpen = false,
+}: NextRoleDialogProps) => {
+  const intl = useIntl();
+  const [isOpen, setOpen] = useState<boolean>(defaultOpen);
+  const nextRole = getFragment(NextRoleDialog_Fragment, nextRoleDialogQuery);
+  const title =
+    `${nextRole.firstName}'s ` +
+    intl.formatMessage({
+      defaultMessage: "next role",
+      id: "CvIcP7",
```

Comment:
Name fallback here too

Reply:
Fixed in [Combine messages for next role dialog title](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13300/commits/b3730d34c9d1939c1939321fb5bcce3e2ada9b2e)